src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,24 +14,28 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
+    public static int nextSequenceNumber = 0;
```

Comment:
Although this works, it's more intuitive to initialize `nextSequenceNumber` to `1` and increment it *after* it has been assigned. As the variable name has the word `next`, it should hold the sequence number for the next Person object, not the most recent one.